lib/Dialect/FIRRTL/Transforms/PrefixModules.cpp

```diff
@@ -227,6 +247,8 @@ void PrefixModulesPass::renameModuleBody(std::string prefix, FModuleOp module) {
         nlaTable->updateModuleInNLA(nla, oldModName, newTarget);
 
       instanceOp.setModuleNameAttr(FlatSymbolRefAttr::get(context, newTarget));
+    } else {
+      replacer.replaceElementsIn(op);
```

Comment:
Why is this "else" -- because we know that MemOp/InstanceOp won't have such refs on them?